bundles/org.openhab.binding.semsportal/README.md

```diff
@@ -0,0 +1,45 @@
+# SEMSPortal Binding
+
+This binding can help you include statistics of your SEMS / GoodWe solar panel insallation into openHAB. It is a readonly connection that maps collected parameters to openhab channels. It provides current, day, month and total yields, as well as some income statistics if you have configured these in the SEMS portal. It requires a power station that is connected through the internet to the SEMS portal.
+
+## Supported Things
+
+There is only one thing type supported in this binding, which is the powerStation. You need to provide the identifying UUID of the solar system in your account yourself. If you have multiple powerstations that report to the same account, you can configure the same account for each powerSation and only differ the unique id of the station in each Thing.
+
+## Discovery
+
+At this moment there is no discovery possible. Configuration of the powerStation unique id is done by hand in the Thing configuration page.
+
+## Thing Configuration
+
+The configuration of the thing is pretty straight forward. You need to have your power station set up in the SEMS portal, and you need to have an account that is allowed to view the power station data. You should log in at least once in the portal with this account to activate it. 
+
+The thing needs the username and password to connect and retreive the data. It also needs the unique id of the power station. It can be found after you log in to the portal, on the URL. use the value you find in the URL at the x-es (including the -s) in this example: https://www.semsportal.com/powerstation/powerstatussnmin/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
```

Comment:
```suggestion
The thing needs the username and password to connect and retreive the data.
It also needs the unique id of the power station. 
It can be found after you log in to the portal, on the URL.
Use the value you find in the URL at the x-es (including the -s) in this example: https://www.semsportal.com/powerstation/powerstatussnmin/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
```

new line for each and a sentence started with a lowercase u.